lib/screens/play/card/card_pile.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:flutter/material.dart';
+import 'package:treasure_of_the_high_seas/model/game_pile.dart';
+
+class CardPile extends StatelessWidget {
+  final int pileSize;
+  final GamePile pileType;
+
+  CardPile(this.pileSize, this.pileType);
+
+  @override
+  Widget build(BuildContext context) {
+
+    // TODO - Allow clicking discard pile? final enabled = pileType == GamePile.DISCARD ?? false;
+    final enabled = false;
+
+    final shape = RoundedRectangleBorder(
+      side: BorderSide(color: Colors.black, width: 1),
+      borderRadius: BorderRadius.circular(10),
+    );
+
+    return Center(
+      child:  RaisedButton(
+        shape: shape,
+        onPressed: enabled ? () {
+          // TODO - Allow clicking on discard pile?
+        }
+            : null,
```

Comment:
Maybe we just go with a `Card` for now rather than a disabled button? It'll probably look a bit nicer. We can revisit turning the discard pile into a button later on (at that point it may well become a separate component to the deck, which will never be clickable)